Payments/Operation/03_ManageRefunds.md

```diff
@@ -0,0 +1,77 @@
+[!!LIST (Refunds](../UserInterface/03_Refunds.md)
+
+
+# Monitor refunds
+
+A refund is a repayment of a capture. On the one hand, it is initialized by the trader, for example, if a product is not available and cannot be delivered. On the other hand, it is initialized if a payment and capture transaction has been manually refunded.
+> [Info] This means that you cannot follow up a case with the ID of an authorization or a payments and captures transaction. However, the most payment service providers support an original reference ID that you can use to search for a specific case in the whole payment process.
```

Comment:
s.o.

Reply:
with the **ID of an authorization or a payments and captures transaction**